main.go

```diff
@@ -30,18 +29,19 @@ func main() {
 		Name:  "autoAPI",
 		Usage: "Generate an CRUD api endpoint program automatically!",
 		Action: func(c *cli.Context) error {
-			f, err := yamlParser.Load(c.String("file"))
+			f, err := configFile.LoadYaml(c.String("file"))
```

Comment:
We may match LoadMethod by `-f filename.<TYPE>`.

Reply:
> We may match LoadMethod by `-f filename.<TYPE>`.

Maybe in the future.